debian/nymea-plugin-stiebeleltron.install.in

```diff
@@ -0,0 +1 @@
+usr/lib/@DEB_HOST_MULTIARCH@/nymea/plugins/libnymea_integrationpluginstiebeleltron.so
```

Comment:
Please also install the translation files:

    stiebeleltron/translations/*qm usr/share/nymea/translations/

Reply:
See https://github.com/nymea/nymea-plugins-modbus/commit/e852d3224e7a1e4fd4c03d49bfd2a9ab4afa3463